docs/web-ui.md

```diff
@@ -369,3 +369,44 @@ The web UI includes a Streaming tab if the application uses Spark streaming. Thi
 scheduling delay and processing time for each micro-batch in the data stream, which can be useful
 for troubleshooting the streaming application.
 
+## JDBC/ODBC Server Tab
+We can see this tab when Spark is running as a [distributed SQL engine](sql-distributed-sql-engine.html) and shows information about sessions and SQL jobs submitted.
+
+At the beginning of the page is the server start time.
+
+<p style="text-align: center;">
+  <img src="img/JDBCServer1.png" width="40%" title="JDBC/ODBC Header" alt="JDBC/ODBC Header">
+</p>
+
+The first section contains information of active and finished sessions with some information.
+* User and IP of the connection.
+* Session id link to access to session info.
+* Start time, finish time and duration of the session.
+* Total jobs submitted in this session.
+
+<p style="text-align: center;">
+  <img src="img/JDBCServer2.png" title="JDBC/ODBC sessions" alt="JDBC/ODBC sessions">
+</p>
+
+The second section has the SQL statistics of the submitted  jobs.
+* **User** that submit the job.
+* **Job id** link to [jobs tab](web-ui.html#jobs-tab).
+* **Group id** of the query that group all jobs together. An application can cancel all running jobs using this group id.
+* **Start time** of the process.
+* **Finish time** of the execution, before fetching the results.
+* **Close time** of the process after fetching the results.
+* **Execution time** is the difference between finish time and start time.
+* **Duration time** is the difference between close time and start time.
+* **Statement** is the SQL statement executed.
```

Comment:
Maybe `SQL statement` -> `SQL statement or operation` being executed?
Since recently, this may not only be the SQL statement, but e.g. an operation like "Listing tables" or "Listing schemas"